components/Button/button.html

```diff
@@ -0,0 +1,107 @@
+          have added different sizes to make the button depending on the
+          view-port.
+        </p>
+
+        <div class="button-container">
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-df btn-color-df">Button</button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-sm btn-color-df">Button</button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-md btn-color-df">Button</button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-lg btn-color-df">Button</button>
+          </a>
+        </div>
+
+        <pre class="code-snippet">
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-df btn-color-df">Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-sm btn-color-df"&gt;Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-md btn-color-df"&gt;Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-lg btn-color-df"&lt;Button&lt;/button&gt;
+          &lt;/a&gt;
+        </pre>
+
+        <hr />
+
+        <h2 class="heading">Different Usage of buttons</h2>
+
+        <div class="button-container">
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-sm btn-color-df">Default</button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-sm btn-color-danger">Danger</button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-sm btn-color-secondary">
+              Secondary
+            </button>
+          </a>
+          <a class="hyper-link" href="#">
+            <button class="btn btn-size-sm btn-color-success">Success</button>
+          </a>
+        </div>
+
+        <pre class="code-snippet">
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-sm btn-color-df">Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-sm btn-color-danger"&gt;Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-sm btn-color-secondary"&gt;Button&lt;/button&gt;
+          &lt;/a&gt;
+          &lt;a class="hyper-link" href="#"&gt;
+           &lt;button class="btn btn-size-sm btn-color-success"&lt;Button&lt;/button&gt;
+          &lt;/a&gt;
+        </pre>
+
+        
+    </main>
+    <script
+      src="https://kit.fontawesome.com/daef01c36c.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
You can import this directly within CSS, by grabbing FontAwesome CDN link from website and pasting in your index.css

Reply:
Thanks for the input didn't knew this.

Reply:
Have imported this link in css file.